playbooks/roles/capsule/tasks/install.yml

```diff
@@ -31,7 +31,7 @@
   register: pulp_oauth_secret
 
 - name: 'Generate Certs'
-  command: capsule-certs-generate --capsule-fqdn {{ ansible_nodename }} --certs-tar {{ capsule_certs_tar }} --disable-system-checks
+  command: capsule-certs-generate --capsule-fqdn {{ ansible_nodename }} --certs-tar {{ capsule_certs_tar }}
```

Comment:
Maybe I saw this with the broken capsule-certs-generate where INSTALLER_DIR was not being properly set by the installer. This should still be merged as the command shouldn't require this to be set, and instead we should update the script/installer to handle this properly.